docs/advanced-ai/examples/vector-store-website.md

```diff
@@ -0,0 +1,29 @@
+---
+contentType: howto
+title: Populate a Pinecone vector database from a website
+description: Scrape a website, load the data into Pinecone, then query it using a chat workflow.
+workflowFile: advanced-ai/examples/populate_a_pinecone_vector_database_from_a_website.json
+---
+
+# Populate a Pinecone vector database from a website
+
+Use n8n to scrape a website, load the data into Pinecone, then query it using a chat workflow. This workflow uses the [HTTP node](/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/) to get website data, improves the data using the [HTML node](/integrations/builtin/core-nodes/n8n-nodes-base.html/), then uses the [Pinecone Vector Store node](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstorepinecone/) to send it to Pinecone. 
```

Comment:
Nitpicking, but `improves the data` sounds a bit vague to me. Perhaps `extracting the relevant content` or something similar would be more fitting? 🤔